qemu/tests/blockdev_mirror_qemuio_ready_job.py

```diff
@@ -0,0 +1,62 @@
+from avocado.utils import process
+
+from virttest import utils_misc
+
+from provider.blockdev_mirror_nowait import BlockdevMirrorNowaitTest
+from provider.job_utils import get_event_by_condition
+
+
+class BlockdevMirrorQemuioReadyjob(BlockdevMirrorNowaitTest):
+    """
+    Qemuio source image when mirror job reach ready status
+    """
+
+    def qemuio_source_image(self):
+        tag = self._source_images[0]
+        image_params = self.params.object_params(tag)
+        img_format = image_params.get("image_format")
+        image = self.source_disk_define_by_params(image_params, tag)
```

Comment:
just fyi.
you can use self.params instead of image specific param object 'image_params', source_disk_define_by_params can take care of that. 
Yours is OK for me